src/Adapter/Cart/CartPresenter.php

```diff
@@ -213,6 +213,7 @@ public function present($cart)
         $products = $this->addCustomizedData($products, $cart);
         $subtotals = [];
 
+        $productsTotalExcludingTax = $cart->getOrderTotal(false, Cart::ONLY_PRODUCTS);
         $total_excluding_tax = $cart->getOrderTotal(false);
```

Comment:
yes, now it's camelCase. We will change all old snake case legacy code, every time we can, smoothly.